scripts/rpc-maas-tool.py

```diff
@@ -668,18 +668,19 @@ def _get_failed_checks(self, checks=None):
                     checks.append(check)
         for check in checks:
                 validation_error = ""
+                completed = False
                 try:
                     result = self.rpcm.conn.test_existing_check(check)
+                    status = result[0]['status']
+                    completed = result[0]['available']
+                    check.state = (" Completed:%(completed)s Status:%(status)s"
+                                   % {'completed': completed,
+                                      'status': status})
                 except rackspace.RackspaceMonitoringValidationError as e:
                     validation_error = " Validation Error: {s}:".format(
                         s=e.message
                     )
-                    break
 
```

Comment:
Instead of moving the following three lines, my preference would be to use `else` to make it clearer which code might throw the exception and what happens whether or not the exception is raised. E.g.

```python
try:
    result = self.rpcm.conn.test_existing_check(check)
except rackspace.RackspaceMonitoringValidationError as e:
    validation_error = " Validation Error: {s}:".format(
        s=e.message
    )
    completed = False
else:
    validation_error = ""
    status = result[0]['status']
    completed = result[0]['available']
    check.state = (" Completed:%(completed)s Status:%(status)s"
                            % {'completed': completed, 'status': status})
```